src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -234,7 +234,7 @@ object Parsers {
         syntaxErrorOrIncomplete(expectedMsg(token))
       }
       if (in.token == token) in.nextToken()
-      in.offset
+      offset
```

Comment:
Yes, good catch. It should be `offset`. Can you add to the doc comment a clause which says "@return   The offset at the start of the token to accept"? Thanks!